lib/nghttp2_callbacks.h

```diff
@@ -120,6 +120,9 @@ struct nghttp2_session_callbacks {
   nghttp2_on_extension_chunk_recv_callback on_extension_chunk_recv_callback;
   nghttp2_error_callback error_callback;
   nghttp2_error_callback2 error_callback2;
+
+  /* Memory allocator */
```

Comment:
Seems like this should be
```
/**
 * Memory allocator
 */
```
and without a linebreak, for consistency with the rest of the fields.